task/prefetch-dependencies/0.2/prefetch-dependencies.yaml

```diff
@@ -356,6 +356,9 @@ spec:
       --for-output-dir=/cachi2/output \
       --output "$(workspaces.source.path)/cachi2/prefetch.env"
 
+      # backwards compatibility with legacy cachi2.env
```

Comment:
@chmeliik @mmorhun what would be a reasonable time frame to remove compatibility hacks like ^this one?